charts/opentelemetry-kube-stack/values.yaml

```diff
@@ -21,6 +26,11 @@ opentelemetry-operator:
     # Setting this allows for an installation of the otel operator at the same time as the custom resources it manages.
     failurePolicy: "Ignore"
 
+  # This is disabled by default, as doing so creates a race condition with helm.
+  # https://github.com/open-telemetry/opentelemetry-helm-charts/issues/677
```

Comment:
We should add that we do not recommend users never set this value to true when using the OKS chart.